Header header
float64 max_val
float64 min_val
